src/gh.ts

```diff
@@ -371,6 +371,53 @@ const completionSpec: Fig.Spec = {
         },
       ],
     },
+    {
+      name: "browse",
+      description: "Open the repository in the browser",
+      args: {
+        name: "[pr | issue | path[:line]",
+        generators: ghGenerators.listPR,
```

Comment:
I'm fine with this as long as this also gets added:
```suggestion
        generators: ghGenerators.listPR,
        suggestCurrentToken: true,
```